recipes/default.rb

```diff
@@ -51,6 +51,16 @@
   nn_https_address = private_recipe_ip("hops", "nn") + ":#{node['hops']['nn']['https_port']}"
 end
 
+if "#{node['hops']['dn']['cloud_data_dir']}".include? ","
+  raise "A datanode cannot have more than one cloud storage"
+end 
+
+if node['hops']['enable_cloud_storage'].eql? "true"
```

Comment:
It's better if you used `casecmp` like https://github.com/logicalclocks/hops-hadoop-chef/blob/master/recipes/default.rb#L276 to do a case insensitive comparison.

Reply:
fxd